### PR TITLE
Include architecture of legacy and upgraded CKAN

### PR DESCRIPTION
**Pull request by brucebolt**

### CODE REVIEW

File: source/manual/data-gov-uk-architecture.html.md [hunk 1]
```diff
@@ -66,6 +68,19 @@ cf env publish-data-beta-production
...
+
+User authentication is handled within Drupal, which sits alongside the highly customised CKAN installation.
+
+Legacy CKAN will be retired when the CKAN upgrade is launched.
+
+## [CKAN (ckanext-datagovuk)][ckan]
+
+This is a plugin to CKAN 2.7.3, as an interim solution before [Publish] is launched. Drupal has been removed and user authentication is handled wholly within CKAN.
+
+TODO: Details about staging/production environments
```
Comment (benthorner) [inline source/manual/data-gov-uk-architecture.html.md]: Is this going to be a separate PR?

Comment (brucebolt) [reply source/manual/data-gov-uk-architecture.html.md]: Yes, I'll update the docs with the staging/prod environments when they have been fully set up

File: source/manual/data-gov-uk-architecture.html.md [hunk 2]
```diff
@@ -66,6 +68,19 @@ cf env publish-data-beta-production
...
+[Legacy CKAN][legacy-ckan] is an extension to CKAN 2.2b and is provisioned on Bytemark hosting.  The following servers are in use:
+
+* Staging: `co-prod2.dh.bytemark.co.uk`
+* Production: `co-prod3.dh.bytemark.co.uk`
+
+User authentication is handled within Drupal, which sits alongside the highly customised CKAN installation.
+
+Legacy CKAN will be retired when the CKAN upgrade is launched.
+
+## [CKAN (ckanext-datagovuk)][ckan]
```
Comment (benthorner) [inline source/manual/data-gov-uk-architecture.html.md]: Ah, OK. In which case, you could lose this section for now, as it's really just the concrete pointers that are useful here - like you did for the legacy ones.
